CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 Changelog
 ===
 
+# 1.1
+
+Release date: Aug 12, 2018
+
+Enhancement:
```

Comment:
Test automation is not a user-visible enhancement. It would be prefereable to move it to a separate changelog group or maybe hide it from the changelog

Reply:
@oleg-nenashev I moved them to the "Test automation" changelog group, WDYT?